lib/generators/react_on_rails/templates/lib/tasks/linters.rake

```diff
@@ -0,0 +1,71 @@
+if %w(development test).include? Rails.env
+  namespace :lint do
+    # require "rubocop/rake_task"
+    # require "slim_lint/rake_task"
+    require "scss_lint/rake_task"
```

Comment:
@aaronvb @derencius I'm OK to adding the linter gems together.

I like @derencius suggested catch block.